amp/repository/aim/view/viewActivityDifferences.jsp

```diff
@@ -90,7 +99,50 @@
 	            		</div>
 	        		</td>
 				</tr>
-				
+
+					<%-- Iterate through the list of output collections for compareAll method... --%>

```

Comment:
also try to reuse the code, not to duplicate the code that already shows the differences